sw/device/silicon_creator/mask_rom/e2e_tests/mask_rom_e2e_testplan.hjson

```diff
@@ -0,0 +1,422 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Testplan TODOs:
+// - TODO(lowRISC/opentitan#10791): Add tests across lifecycle states
+//   - Test functionality that exists in some LC states but fails in others
+//   - This could make use of test tags (#10536)
+// - TODO(lowRISC/opentitan#10792): Add tests for behavior during manufacture
+//   - Bringup keys for manufacturing and RMA
+
+{
+  name: "mask_rom_e2e"
+
+  testpoints: [
+    // Bootup without ROM_EXT
+    {
+      name: mask_rom_e2e_bootup_no_rom_ext
+      desc: '''Verify device fails to boot with no ROM_EXT.
+
+            - Test runner loads the Mask ROM and no ROM_EXT.
+            - Attempt to boot.
+            - Verify that the chip indicates the correct boot fault.
+            '''
+      tags: ["verilator", "dv", "fpga", "silicon"]
```

Comment:
I will also suggest tagging all of these testpoints with `mask_rom` for ease of filtering.

Reply:
fixed.